src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -0,0 +1,2271 @@
+			}
+
+			if (!inventoryRowList.isEmpty()) {
+				for (MedicalInventoryRow invRow : inventoryRowList) {
+					addMedInRowInInventorySearchList(invRow);
+					if (!includeAll && invRow.getId() == 0) {
+						inventoryRowListAdded.add(invRow);
+					}
+				}
+			} else {
+				if (!mode.equals("new")) {
+					MessageDialog.info(null, "angal.invetory.notdataforthatfilter.msg");
+				}
+			}
+		}
+
+		public Class< ? > getColumnClass(int c) {
+			return columnsClasses[c];
+		}
+
+		public int getRowCount() {
+			if (inventoryRowSearchList == null) {
+				return 0;
+			}
+			return inventoryRowSearchList.size();
+		}
+
+		public String getColumnName(int c) {
+			return columsNames[c];
+		}
+
+		public int getColumnCount() {
+			return columsNames.length;
+		}
+
+		public Object getValueAt(int r, int c) {
+			if (r < inventoryRowSearchList.size()) {
+				MedicalInventoryRow medInvtRow = inventoryRowSearchList.get(r);
+				if (c == -1) {
+					return medInvtRow;
+				} else if (c == 0) {
+					return medInvtRow.getId();
+				} else if (c == 1) {
+					return medInvtRow.getMedical() == null ? "" : medInvtRow.getMedical().getProdCode();
+				} else if (c == 2) {
+					return medInvtRow.getMedical() == null ? "" : medInvtRow.getMedical().getDescription();
+				} else if (c == 3) {
+					if (medInvtRow.getLot() == null || medInvtRow.isNewLot()) {
+						return "N";
+					}
+					return "";
+				} else if (c == 4) {
+					if (medInvtRow.getLot() == null) {
+						return "";
+					}
+					return medInvtRow.getLot().getCode().equals("") ? "AUTO" : medInvtRow.getLot().getCode();
+				} else if (c == 5) {
+					if (medInvtRow.getLot() != null) {
+						if (medInvtRow.getLot().getDueDate() != null) {
+							return medInvtRow.getLot().getDueDate().format(DATE_FORMATTER);
+						}
+					}
+					return "";
+				} else if (c == 6) {
+					Double dblVal = medInvtRow.getTheoreticQty();
+					return dblVal.intValue();
+				} else if (c == 7) {
+					Double dblValue = medInvtRow.getRealQty();
+					return dblValue.intValue();
+				} else if (c == 8) {
+					if (medInvtRow.getLot() != null) {
+						if (medInvtRow.getLot().getCost() != null) {
+							medInvtRow.setTotal(medInvtRow.getLot().getCost().multiply(BigDecimal.valueOf(medInvtRow.getRealQty())));
+							return medInvtRow.getLot().getCost();
+						}
+					}
+					return BigDecimal.ZERO;
+				} else if (c == 9) {
+					if (medInvtRow.getLot() != null) {
+						if (medInvtRow.getLot().getCost() != null) {
+							return medInvtRow.getTotal();
+						}
+					}
+					return BigDecimal.ZERO;
+				}
+			}
+			return null;
+		}
+
+		@Override
+		public void setValueAt(Object value, int r, int c) {
+			if (r < inventoryRowSearchList.size()) {
+				MedicalInventoryRow invRow = inventoryRowSearchList.get(r);
+				if (c == 7) {
+					Double intValue = 0.0;
+					if (value != null) {
+						try {
+							intValue = Double.parseDouble(value.toString());
+						} catch (NumberFormatException e) {
+							intValue = 0.0;
```

Comment:
remove this statement, `intValue` is a local variable so this assignment is useless